internal/plan/type.go

```diff
@@ -1,21 +1,23 @@
 package plan
 
+type TestCaseFormat string
+
+const (
+	TestCaseFormatFile    TestCaseFormat = "file"
+	TestCaseFormatExample TestCaseFormat = "example"
+)
+
 // TestCase represents a single test case.
 type TestCase struct {
-	Path              string `json:"path"`
-	EstimatedDuration *int   `json:"estimated_duration"`
-}
-
-// Tests represents a set of tests.
-type Tests struct {
-	Cases  []TestCase `json:"cases"`
-	Format string     `json:"format"`
+	Path              string         `json:"path"`
+	EstimatedDuration *int           `json:"estimated_duration,omitempty"`
```

Comment:
I added `omitempty` option to prevent sending the request with these empty attributes, as they are optional.